lib/ffmpeg/movie.rb

```diff
@@ -16,7 +16,6 @@ class Movie
     UNSUPPORTED_CODEC_PATTERN = /^Unsupported codec with id (\d+) for input stream (\d+)$/
 
     @has_dynamic_resolution = false
-    @requires_pre_encode = false
```

Comment:
nit: It might be nice to keep this still even though we're assigning it's value to `nil` to indicate it _should_ be a variable accessible in the API / explicitly nil until set 🤔